docs/Components.md

```diff
@@ -0,0 +1,56 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Anatomy of a calico-node container
+
+`calico/node` can be regarded as a helper container that bundles together the 
+various components required for networking containers with Calico.  The 
+calico/node Docker image (the code for which is in this repo) is comprised of:
```

Comment:
"the code for which is in this repo"

I don't think this is worded correctly.  It kind of sounds like Felix, BIRD, and confd are in this repo.  It might be clearer to leave this part out or say that the code to create the docker image is in here.